### PR TITLE
Homework6: Shevchuk Aleksei

### PR DESCRIPTION
**Pull request by alex-shef**

### CODE REVIEW

File: Homework6/task2.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    def decorator(function):
+        def wrapper(*args):
+            nonlocal num
+            while num:
+                try:
+                    result = function(*args)
+                    return result
+                except Exception as exception:
+                    num -= 1
+                    return exception
```
Comment (belovmd) [inline Homework6/task2.py:17]: it shouldn't return exception, it should rerun it

File: Homework6/task2.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            else:
+                try:
+                    raise TooManyErrors
```
Comment (belovmd) [inline Homework6/task2.py:20]: why do we need to raise it and catch in the next line?

File: Homework6/task1.py [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+from time import sleep
+
+
+class Person(object):
+    antagonist = None
+    hp = 100
+    max_HP = hp
+    simple_damage = [18, 25]
+    special_damage = [10, 35]
+    first_aid_kit = [15, 25]
```
Comment (belovmd) [inline Homework6/task1.py:17]: what doesn all these numbers mean?

Comment (belovmd) [reply Homework6/task1.py:17]: it seems aid can be separate class

**Review by belovmd — COMMENTED** [review]
wrong decorator task. 
pretty simple class
6,5

File: Homework6/task1.py (hunk 3 above)
Comment (alex-shef) [reply Homework6/task1.py:17]: these are the limits of change hp

**Pull request closed without merging**